client/scripts/legic.lua

```diff
@@ -217,8 +217,8 @@ end
 local currency = {
   ["03d2"]="EUR",
   ["0348"]="USD",
-  ["033A"]="GBP",
-  ["02F4"]="CHF"
+  ["033a"]="GBP",
+  ["02f4"]="CHF"
 }
```

Comment:
hm, you set some values to UPPER.. but not all.  So you need to makes sure all calls to this lua-table uses UPPER or LOWER string values and match the lua-table entries accordingly.